packages/calypso-babel-config/presets/default.js

```diff
@@ -27,7 +27,13 @@ module.exports = ( api, opts ) => ( {
 		],
 		[
 			require.resolve( '@babel/preset-react' ),
-			{ runtime: 'automatic', importSource: '@emotion/react' },
+			{
+				runtime: 'automatic',
+				importSource:
+					typeof process.env.IMPORT_SOURCE !== 'undefined'
```

Comment:
Should we document this new environment variable in the package README?

Reply:
Definitely, I'll add it.